### PR TITLE
add registrar_host to api deployment

### PR DESCRIPTION
**Pull request by Zacharis278**

### CODE REVIEW

File: edxpipelines/pipelines/api_deploy.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ def install_pipelines(configurator, config):
             'BURST_LIMIT': config['aws']['burst_limit'],
             'EDXAPP_HOST': config['upstream_origins']['edxapp'],
             'CATALOG_HOST': config['upstream_origins']['catalog'],
+            'REGISTRAR_HOST': config['upstream_origins']['registrar'],
```
Comment (bderusha) [inline edxpipelines/pipelines/api_deploy.py]: Should this be `registrar_api`?

Comment (bderusha) [reply edxpipelines/pipelines/api_deploy.py]: If so we should also update the var name to include API in it for consistency

Comment (Zacharis278) [reply edxpipelines/pipelines/api_deploy.py]: it could be?  I just did host since lms/catalog did it that way

Comment (Zacharis278) [reply edxpipelines/pipelines/api_deploy.py]: oic now... on it

File: test-config.yml [hunk 2]
```diff
@@ -22,6 +22,7 @@ global-config:
         edxapp: dummy_edxapp
         catalog: dummy_catalog
         analytics_api: dummy_analytics_api
+        registrar_api: dummy_registrar
```
Comment (bderusha) [inline test-config.yml]: NIT: `dummy_registrar_api` for consistency

**Pull request merged**